app/src/org/commcare/android/models/logic/FormRecordProcessor.java

```diff
@@ -61,12 +63,20 @@ public FormRecord process(FormRecord record)
             FormUploadUtil.getDecryptCipher((new SecretKeySpec(record.getAesKey(), "AES")));
         InputStream is = new CipherInputStream(new FileInputStream(f), decrypter);
 
-        AndroidTransactionParserFactory factory = new AndroidTransactionParserFactory(c, null);
-
-        factory.initCaseParser();
-        factory.initStockParser();
```

Comment:
These calls occur already when the `AndroidTransactionParserFactory` calls its super method https://github.com/dimagi/commcare/blob/master/core/org/commcare/core/parse/CommCareTransactionParserFactory.java#L54-L60